contrib/coredns/policy/policy.go

```diff
@@ -119,6 +119,52 @@ func (p *PolicyMiddleware) getEDNS0Attrs(r *dns.Msg) ([]*pb.Attribute, bool) {
 	return attrs, foundSourceIP
 }
 
+
+type LocalResponseWriter struct {
+	dns.ResponseWriter
```

Comment:
dns.ResponseWriter interface is required as NextOfFailure requires dns.ResponseWriter parameter

func NextOrFailure(name string, next Handler, ctx context.Context, w dns.ResponseWriter, r *dns.Msg) (int, error) {